README.md

```diff
@@ -978,20 +979,47 @@ $ nf-core modules install
     nf-core/tools version 2.0
 
 ? Tool name: cat/fastq
-? Select 'cat/fastq' version: Rename software/ directory to modules/ to re-organise module structure ...truncated...
 INFO     Installing cat/fastq
 INFO     Downloaded 3 files to ./modules/nf-core/modules/cat/fastq
 ```
 
 You can pass the module name as an optional argument to `nf-core modules install` instead of using the cli prompt, eg: `nf-core modules install fastqc`.
 
-There are five flags that you can use with this command:
+There are four flags that you can use with this command:
 
 * `--dir <pipeline_dir>`: Specify a pipeline directory other than the current working directory.
-* `--latest`: Install the latest version of the module instead of specifying the version using the cli prompt.
+* `--prompt`: Select the module version using a cli prompt.
```

Comment:
Why do we need this command as opposed to using `--sha`?

Reply:
`--prompt` shows a list of versions from which the user can select the correct one. `--sha` accepts a commit SHA as an argument and installs the module at that commit without any prompts, which we need for running the command non-interactively.

Reply:
I though the default behaviour if you run `nf-core modules update` is to go via prompts? i.e. without any command-line flags.